apl/tabular-operators/project-keep-operator.mdx

```diff
@@ -1,46 +1,155 @@
+You can use `project-keep` when you need to focus on particular data points, such as in log analysis, security event monitoring, or extracting key fields from traces.
 
-## Syntax
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+In Splunk SPL, the `table` command performs a similar task to APL’s `project-keep`. It selects only the columns you specify and excludes any others.
+
+<CodeGroup>
+```splunk Splunk example
+index=main | table _time, status, uri
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| project-keep _time, status, uri
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+In ANSI SQL, the `SELECT` statement combined with column names performs a task similar to `project-keep` in APL. Both allow you to specify which columns to retrieve from the dataset.
+
+<CodeGroup>
+```sql SQL example
+SELECT _time, status, uri FROM sample_http_logs
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| project-keep _time, status, uri
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
 
 ```kusto
-| project-keep FieldName1, FieldName2, ...
+| project-keep ColumnName1, ColumnName2, ...
 ```
 
-## Arguments
+### Parameters
+
+- `ColumnName`: The column you want to keep in the result set.
+
+### Returns
 
-| **name**  | **type** | **description**                                    |
-| --------- | -------- | -------------------------------------------------- |
-| Field Name | **string** | The name of the field to be kept in the output. |
+`project-keep` returns a dataset with only the specified columns. All other columns are removed from the output.
 
-## Returns
+## Use case examples
 
-A table with fields that are named as arguments. Contains the same number of rows as the input table.
+<Tabs>
+<Tab title="Log analysis">
 
-## Examples
+For log analysis, you might want to keep only the columns that are relevant to investigating HTTP requests.
+
+**Query**
 
 ```kusto
-['http-logs']
-| project-keep  ['geo.city'], ['geo.country'], method
+['sample-http-logs'] 
+| project-keep _time, status, uri, method, req_duration_ms
 ```
 
-[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%5Cn%7C%20project-keep%20%20%5B%27geo.city%27%5D%2C%20%5B%27geo.country%27%5D%2C%20method%22%2C%22queryOptions%22%3A%7B%22quickRange%22%3A%2230d%22%7D%7D)
+[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%20%7C%20project-keep%20_time%2C%20status%2C%20uri%2C%20method%2C%20req_duration_ms%22%7D)
```

Comment:
![image](https://github.com/user-attachments/assets/319bdd39-9782-4db7-aa6c-f3523fa70ce7)

Reply:
I don't have an issue with this
<img width="604" alt="image" src="https://github.com/user-attachments/assets/8f2e7183-64b0-48c7-8a69-7d3283984eba">